app/apiProvider.js

```diff
@@ -44,4 +44,12 @@ export class ApiProvider {
     logger.info(`api called: [GET] ${endpoint}`);
     return axios.get(endpoint);
   }
+
+  async postSelectedCapabilities(selectedCapabilitiesJson) {
+    const endpoint = `${this.apiHost}/api/v1/Solutions`;
+    logger.info(`api called: [POST] ${endpoint}: ${JSON.stringify(selectedCapabilitiesJson)}`);
+    // TODO: change endpoint below when API work is done
+    // return axios.post(endpoint, selectedCapabilities);
```

Comment:
Should we leave this like this or change it back. It would mean creating a PR to undo this line once the API work is done and also means our task is not complete

Reply:
but it shows it works?